bugbug/bugzilla.py

```diff
@@ -93,50 +94,84 @@ def bughandler(bug):
     return all_ids
 
 
-def get(ids_or_query):
-    new_bugs = {}
+def get(bug_ids: Union[List[int], Set[int]]):
+    """Function to retrieve Bug Information including history, attachment, comments using Bugzilla REST API.
 
-    def bughandler(bug):
-        bug_id = int(bug["id"])
-
-        if bug_id not in new_bugs:
-            new_bugs[bug_id] = dict()
-
-        new_bugs[bug_id].update(bug)
-
-    def commenthandler(bug, bug_id):
-        bug_id = int(bug_id)
-
-        if bug_id not in new_bugs:
-            new_bugs[bug_id] = dict()
+    :param bug_ids: find bug information for these `bug_ids`
+    :type bug_ids: list of integers or str or int
+    :return: dict with key as `id`(int) of a bug,  and values as bug_information
+    :rtype: dict
+    """
 
-        new_bugs[bug_id]["comments"] = bug["comments"]
+    def filter_keys(array_of_dict_values, required_fields):
+        # This will mutates array_of_dict_values values in place
+        for a_ind, a_value in enumerate(array_of_dict_values):
+            a_value = {
+                a_field: a_value[a_field]
+                for a_field in list(a_value.keys())
+                if a_field in required_fields
+            }
+            array_of_dict_values[a_ind] = a_value
+        return array_of_dict_values
+
+    def cleanup_fields_in_batch_of_bugs(batch_of_bugs):
```

Comment:
Can you give me a list of the attributes that are cleaned up?

Reply:
The Comments sub fields - bug_id, attachment_id, raw_text, creator, time, creation_time, is_private, is_markdown are Cleaned-Up.

Reply:
If those are the only ones, I think we can avoid the cleanup altogether (so the code will be simpler).

Reply:
Removed the codes for cleanup activity of Comments fields. Fixed